### ISSUE TITLE
Search alt text for commas as a part of the pre-processing instead of the post-processing

### ISSUE THREAD

**Issue by cgrandin** [pbs-assess/csasdowntex#289]
**Tagging branch**

Commas cannot be used in alt text, it simply is impossible due to limitations of the LaTex tagPDF package. To mitigate this, currently the alt text is searched one-by-one as it is processed in the post-processing and an error is thrown if a comma is found. This causes a huge time waste as you have to wait for the entire document to build to catch only one violation at a time. There are typically many violations, so this can take a long time of waiting.

Possible solutions:
1. Have the parser remove the commas and issue a warning instead, leaving the processing in the post-processing step (super easy and fast to change)
2. Write a routine that is run during pre-processing which checks for commas in alt text and gives you a list of the ones that contain a comma, throwing an error straight away (more significant time coding but nicer for output of alt text)

**Comment by cgrandin** [comment]
Problem with issuing a warning during post-processing is it is not printed at all. Even when using `sink()`, it cannot be printed to the screen.

**Comment by seananderson** [comment]
CSAP told me acceptable alt text is 'Figure 1', 'Figure 2', etc. If we auto do this, we'll also solve the issue with (1) people having to supply their own alt text and (2) this comma problem.

**Comment by cgrandin** [comment]
No longer an issue, just replace commas. New code will set alt text to be the figure number anyways
